sdk/program/src/vote/state/mod.rs

```diff
@@ -789,6 +567,101 @@ impl VoteState {
     }
 }
 
+pub mod serde_compact_vote_state {
+    use {
+        super::*,
+        crate::{
+            clock::{Slot, UnixTimestamp},
+            serde_varint, short_vec,
+            vote::state::Lockout,
+        },
+        serde::{Deserialize, Deserializer, Serialize, Serializer},
+    };
+
+    #[derive(Deserialize, Serialize, AbiExample)]
+    struct CompactLockout {
+        #[serde(with = "serde_varint")]
+        offset: Slot,
+        #[serde(with = "serde_varint")]
+        confirmation_count: u32,
```

Comment:
can be u8

Reply:
done.

Reply:
Was copied over from the original `Vote` which defined `confirmation_count` as `u32`. @carllin was there a specific reason for that?